starcheck/src/starcheck

```diff
@@ -3,12 +3,12 @@
 unset LD_LIBRARY_PATH
 unset SYBASE
 unset PYTHONPATH
-SKA_bin=`dirname $0`
-eval `${SKA_bin}/flt_envs -shell sh`
+eval `${SKA}/bin/flt_envs -shell sh`
 export PYTHONHOME=$SKA_ARCH_OS
+export SCDIR=${SKA_ARCH_OS}/lib/python2.7/site-packages/starcheck
```

Comment:
Realistically this is fine.  But I can also see something a little more generic like:
```
export SCDIR=`python -c 'import starcheck, os; print(os.path.dirname(starcheck.__file__))'`
```
EDIT:
```
export STARCHECK=`python -c 'import starcheck; print(starcheck.__path__[0])'`
```